internal/worker/kv_status.go

```diff
@@ -111,6 +111,7 @@ const (
 	indeterminate           // we got an error in the process of making the check
 )
 
+//nolint:gocyclo // fun fact: there are no peer-reviewed studies of software quality that support style checking as a benefit
```

Comment:
If there was one, it would be an interesting read for sure. 
Assuming automated style/linter checks impact readability in in a positive manner - and more readability in turn means less likelihood of bugs.. I guess there is a point of diminishing returns as well

Reply:
And here I thought no one would read my snarky `nolint` comments. 😅